.github/workflows/integration.yml

```diff
@@ -0,0 +1,35 @@
+name: Integration
+
+on:
+  push:
+    branches: [master, main, force_test, release-*, staging, trying]
+  pull_request:
+    branches: [master, main, force_test, release-*]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        test-setups:
+        - {name: "TimescaleDB 2.x without Promscale extension (pg13)", image: "timescale/timescaledb:latest-pg13"}
+        - {name: "TimescaleDB 2.x with Promscale extension (pg13)", image: "timescaledev/promscale-extension:latest-ts2-pg13"}
+        - {name: "TimescaleDB 2.x with Promscale extension (pg12)", image: "timescaledev/promscale-extension:latest-ts2-pg12"}
+        - {name: "TimescaleDB 1.x with Promscale extension (pg12)", image: "timescaledev/promscale-extension:latest-ts1-pg12"}
+        - {name: "Test against first supported", image: "timescale/timescaledb:1.7.4-pg12"}
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Import environment variables
+      run: cat ".github/env" >> $GITHUB_ENV
+
+    - name: Set up Go ${{ env.golang-version }}
+      uses: actions/setup-go@v2.1.4
+      with:
+        go-version: ${{ env.golang-version }}
+      id: go
+
+    - name: ${{ matrix.test-setups.name }}
+      run: ./scripts/end_to_end_tests.sh ${{ matrix.test-setups.image }}
+      shell: bash
```

Comment:
do we need the equivalent of "e2e results" here to sum up the results so we can put it into branch protection rules

Reply:
I think so.

Reply:
added